### ISSUE TITLE
BottomTabs inserting edges on ios

### ISSUE THREAD

**Issue by thedxrklord** [react-navigation/react-navigation#11127]
### Current behavior

Extra blank space is displayed when using Bottom Tab Navigator

<img width="373" alt="image" src="https://user-images.githubusercontent.com/77444983/209869312-45071fa1-0fce-4ab2-93b7-b3498d967b8f.png">

I saw already some solutions for v5, but they doesn't work for v6
Adding own "SafeAreaVIew" or paddingBottom doesn't help too

### Expected behavior

Is displayed without paddings, if I want - I add safeareaview

### Reproduction

https://github.com/thedxrklord/miykray

### Platform

- [ ] Android
- [X] iOS
- [ ] Web
- [ ] Windows
- [ ] MacOS

### Packages

- [X] @react-navigation/​bottom-tabs
- [ ] @react-navigation/​drawer
- [ ] @react-navigation/​material-bottom-tabs
- [ ] @react-navigation/​material-top-tabs
- [ ] @react-navigation/​stack
- [ ] @react-navigation/​native-stack
- [ ] react-native-tab-view
- [ ] flipper-plugin-react-navigation

### Environment

- [x] I've removed the packages that I don't use

| package                                | version |
| -------------------------------------- | ------- |
| @react-navigation/native               |6.1.1
| @react-navigation/bottom-tabs          |6.5.2
| react-native-safe-area-context          |4.4.1
| react-native                           |0.70.5
| expo                                   |47.0.9

**Comment by cospin** [comment]
Same issue here. Any workarounds?

**Comment by raajnadar** [comment]
Did you try this prop from react native safe area context? https://github.com/th3rdwave/react-native-safe-area-context#edges

**Comment by cospin** [comment]
Thanks @raajnadar! It's not exactly as you say, but that led me to the follow usage, I didn't find the option in the documentation, it would be nice to add it.

```
<Tab.Navigator safeAreaInsets={{bottom: 0}} >
  // ...tabs
</Tab.Navigator>
```

Seems to work as expected, @thedxrklord

**Comment by SohaibKtb** [comment]
@cospin I addedd safeAreaInsets={{bottom: 0}} to the tab navigator but it is not working for material-bottom-tabs, it is only working for bottom-tabs on iPhone 14 pro, thanks

**Comment by roni-castro** [comment]
> @cospin I addedd safeAreaInsets={{bottom: 0}} to the tab navigator but it is not working for material-bottom-tabs, it is only working for bottom-tabs on iPhone 14 pro, thanks

I think this happens probably because in the `material-bottom-tabs` the props [`safeAreaInsets` is harcoded](https://github.com/react-navigation/react-navigation/blob/090c58cd11b82f522fba715e0edc3a5051fa26ed/packages/material-bottom-tabs/src/views/MaterialBottomTabView.tsx#L215) and on [bottom-tabs it is not](https://github.com/react-navigation/react-navigation/blob/e13b4d9341362512ba4bf921a17552f3be8735c1/packages/bottom-tabs/src/views/BottomTabView.tsx#L78-L81)

**Comment by cospin** [comment]
@SohaibKtb seems like this was just fixed on material-bottom-tabs@6.2.13: https://github.com/react-navigation/react-navigation/commit/fbfc9b39db4ac8cb56d0c8f28b0f27b8cda0a508 

I think the issue could be closed.
Although, I would think it's missing to add this usage to the documentation of the bottom and top navigations.

**Comment by VictorioMolina** [comment]
After updating react-native-paper to last version, I am experiencing a similar issue. In this case, the bottom tab has a greater height and icons are vertically aligned to the center on iOS.
